### PR TITLE
New versioning

### PR DESCRIPTION
**Pull request by limor-gs**

### CODE REVIEW

File: packaging/restservice/build.spec [hunk 1]
```diff
@@ -75,7 +75,7 @@ if [ ! -d "/opt/manager/env" ]; then virtualenv /opt/manager/env; fi && \
 /opt/manager/env/bin/pip install --upgrade --force-reinstall --use-wheel --no-index --find-links=/var/wheels/%{name} cloudify-agent --pre && \
 /opt/manager/env/bin/pip install --upgrade --force-reinstall --use-wheel --no-index --find-links=/var/wheels/%{name} ldappy --pre && \
 /opt/manager/env/bin/pip install --upgrade --force-reinstall --use-wheel --no-index --find-links=/var/wheels/%{name} cloudify-rest-service --pre
-if [ "%{PREMIUM}" == "true" ]; then
+if [ "%{REPO}" == "cloudify-versions" ]; then
```
Comment (limor-gs) [inline packaging/restservice/build.spec]: !=

File: packaging/manager/provision.sh [hunk 2]
```diff
@@ -14,48 +14,51 @@ function download_resources() {
...
+export REPO=$3
+export GITHUB_USERNAME=$4
+export GITHUB_PASSWORD=$5
 
-echo "PREMIUM=$PREMIUM"
-if [ "$PREMIUM" == "true" ]; then
-    export AWS_S3_PATH=$AWS_S3_PATH"/"$PREMIUM_FOLDER
+if [ $REPO == "cloudify-versions" ];then
+    #REPO_TAG="master"
+    REPO_TAG="new-versioning"
```
Comment (idanmo) [inline packaging/manager/provision.sh]: In both conditions, REPO_TAG is the same...?

File: packaging/mgmtworker/provision.sh [hunk 3]
```diff
@@ -21,12 +21,28 @@ function build_rpm() {
...
 
 AWS_ACCESS_KEY_ID=$1
 AWS_ACCESS_KEY=$2
+export REPO=$3
+export GITHUB_USERNAME=$4
+export GITHUB_PASSWORD=$5
+
+if [ $REPO == "cloudify-versions" ];then
+    #REPO_TAG="master"
+    REPO_TAG="new-versioning"
```
Comment (idanmo) [inline packaging/mgmtworker/provision.sh]: ?

File: packaging/manager/provision.sh (hunk 2 above)
Comment (limor-gs) [inline packaging/manager/provision.sh]: Only for tests - I will change it before merging

**Pull request merged**